packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -102,7 +102,7 @@ export class FeatureDevController {
                 getLogger().error('processChatItemVotedMessage failed: %s', (e as Error).message)
             })
         })
-        this.chatControllerMessageListeners.followUpClicked.event(data => {
+        this.chatControllerMessageListeners.followUpClicked.event(async data => {
```

Comment:
Can't see an `await` in the block. Any reason this is explicitly declared `async`?

Reply:
leftover, I need to remove this.